internal/testutils/jimmtest/mocks/jimm_group_mock.go

```diff
@@ -55,11 +55,11 @@ func (j *GroupService) GetGroupByName(ctx context.Context, user *openfga.User, n
 	return j.GetGroupByName_(ctx, user, name)
 }
 
-func (j *GroupService) ListGroups(ctx context.Context, user *openfga.User, filters pagination.LimitOffsetPagination) ([]dbmodel.GroupEntry, error) {
+func (j *GroupService) ListGroups(ctx context.Context, user *openfga.User, pagination pagination.LimitOffsetPagination, match string) ([]dbmodel.GroupEntry, error) {
```

Comment:
some places we call it `filter`, other places `match`. we should decide on one of those and use that term everywhere.